src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -62,6 +62,35 @@ public enum DynamicKeywordBodyMode
         Hashtable = 2,
     }
 
+    /// <summary>
+    /// Defines the permissible suffixes for numerical tokens.
+    /// </summary>
+    [Flags]
+    public enum NumberSuffixFlags
```

Comment:
I would only make this public if we exposed the flags in the Token api, which could be nice but isn't important.

Reply:
Roger that~! I'll get on it.

Aside: not sure why, but the Get-WinEvent tests are breaking, and I'm *pretty darn sure* I've not changed anything in that code specifically...

Reply:
Just remove `public` - you don't need `internal`.

Reply:
@vexx32 The `Get-WindEvent` tests are currently just volatile and need fixing -- nothing to do with your changes